cmd/main.go

```diff
@@ -41,12 +41,15 @@ func main() {
 		})
 	})
 
-	namespacesGroup := engine.Group("/namespaces")
+	v1 := engine.Group("/v1")
 	{
-		namespacesGroup.GET("/", routes.ListNamespaces(client, logger))
-		namespacesGroup.GET("/:name", routes.GetNamespace(client, logger))
-		namespacesGroup.POST("/", routes.CreateNamespace(client, logger))
-		namespacesGroup.DELETE("/:name", routes.DeleteNamespace(client, logger))
+		namespacesGroup := v1.Group("/namespaces")
+		{
+			namespacesGroup.GET("/", routes.ListNamespaces(client, logger))
+			namespacesGroup.GET("/:name", routes.GetNamespace(client, logger))
+			namespacesGroup.POST("/", routes.CreateNamespace(client, logger))
+			namespacesGroup.DELETE("/:name", routes.DeleteNamespace(client, logger))
+		}
 	}
```

Comment:
This change exists in another PR as well, no?

Reply:
Yes, I need here it in the tests so I can write tests to execute http requests on `/v1/...` path, it's needed so we won't send another PR to refactor it later.